SPECS-EXTENDED/rubygem-metaclass/rubygem-metaclass.spec

```diff
@@ -0,0 +1,152 @@
+# Generated from metaclass-0.0.1.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name metaclass
+
+
+Summary:        Adds a metaclass method to all Ruby objects
+Name:           rubygem-%{gem_name}
+Version:        0.0.4
+Release:        14%{?dist}
+# https://github.com/floehopper/metaclass/issues/1
```

Comment:
Another one to remove - resolved since 0.0.2.

Reply:
fixed